### DISCUSSION TITLE
owo

### DISCUSSION THREAD

**Discussion by Tanza3D** [Osekai/eclipse#96] [General]
owo

**Comment by minusQuantumNeko** [comment]
Oi

**Reply by Tanza3D** [reply]
what :(

**Reply by EXtremeExploit** [reply]
QUESO AJALJSJASKJLJKLASD

**Reply by Tanza3D** [reply]
cheese mmmm

**Reply by infadel2** [reply]
thanks

**Comment by FujiwaraSatomi** [comment]
Are you planning to move from osekai forum on discord to this github discussion?

**Reply by minusQuantumNeko** [reply]
> Are you planning to move from osekai forum on discord to this github discussion?

Nah, not really. It's just easier to use for ideas, polls and stuff. Plus it's helpful for people who for some reason don't have Discord!

**Reply by FujiwaraSatomi** [reply]
I see
